setup.py

```diff
@@ -1,5 +1,48 @@
+import pip
 import setuptools
 
+def v1_ge_v2(module, version):
+    """return `module.__version__ >= version`"""
+
+    def v2li(v):
+        """`v2li('20.0') == [20, 0]`"""
+        return list(map(int, v.split('.')))
+
+    version = v2li(version)
+    module = v2li(module)
+    # add semver zeros if, e.g., version='20.5.1' and module='20.5'
+    if len(version) > len(module):
+        module += [0] * (len(version) - len(module))
+
+    # Compare version numbers
+    for m, v in zip(module, version):
+        if m > v:
+            # current SEMVER number larger
+            return True
+        elif m == v:
+            # go to next number
+            continue
+        # skip `else` case
+        break
+    else:
+        # Version numbers exactly equal
+        return True
+    # Indeed, `module.__version__ < version`
+    return False
```

Comment:
I know this is rather a complicated logic.

Reply:
Yeah, I think your comments make it readable.